lib/checks/navigation/p-as-heading.js

```diff
@@ -1,4 +1,5 @@
-let siblings = Array.from(node.parentNode.children);
+let parent = node.parentNode;
```

Comment:
This change does not seem necessary

Reply:
ah, a leftover thing from some experimentation I did. you're right–I thought I restored it, but I guess not.